Create oscp.ron

savdbroek opened the pull request:
I added a lens for OSCP URLs

savdbroek committed:
Create oscp.ron

I added a lens for OSCP URLs

a5huynh commented:
Hey @savdbroek, thanks for contributing this lens 🙂. I'll take a look and merge this in

If you don't mind me asking, how did you discover Spyglass? And are there are any features that you'd like to see implemented?

travolin commented:
We have added the lens to the crawl queue for preprocessing

travolin committed:
Update oscp.ron

- Removed spaces from name
- Add label
- Remove extra newlines
- Remove unused is enabled flag

travolin committed:
Update oscp.ron

- Update name to match file
